src/site/content/en/blog/origin-trials/index.md

```diff
@@ -38,21 +38,31 @@ trials, as learnings are incorporated and adjustments are made.
 ## How to register for an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Request a token](https://developers.chrome.com/origintrials/#/trials/active).
-1. [Add the token](http://googlechrome.github.io/OriginTrials/developer-guide.html)
-   to your web pages, using a `<meta>` tag or an `Origin-Trial` HTTP response
-   header.
+1. Request a token by clicking the **Request** button and filling out the form.
```

Comment:
Did you mean "Register" instead of "Request"?

Reply:
Double "request" is bad style indeed. I went for "obtain" now.

Reply:
I meant the button name, not the verb. Sorry I should have been more clear.